### PR TITLE
Updating wiki and ryan's page

### PR DESCRIPTION
**Pull request by ryanstwrt**

### CODE REVIEW

File: users/stewryan/index.md [hunk 1]
```diff
@@ -8,26 +8,32 @@ author: Ryan Stewart
...
-### Publications 
-1. R. Stewart, and T. Palmer, “Metallic Fuel Design Space for Sodium Fast Reactors,” Transactions of the American Nuclear Society, vol. 119, 2018, pp. 1116-1119.
-2. D. LaBrier, R. Stewart, et al., “Design of an In-Pile Experimental Loop for Fast Test Reactor Conditions,” Transactions of the American Nuclear Society, vol. 119, 2018, pp. 1271-1274.
-3. E. Lum, C. Pope, R. Stewart, et al. "Evaluation of Run 138B at Experimental Breeder Reactor II, A Prototypic Liquid Metal Fast Breeder Reactor," International Handbook of Evaluated Reactor Physics Benchmark Experiments, OECD, 2018.
-4. R. Stewart, C. Pope, and E. Ryan, "Fast Spectrum Reactor Fuel Assembly Sensitivity Analysis," Annals of Nuclear Energy, vol. 110, 2017, pp. 1091-1097.
+2. Multi-Objective Optimization
+3. Applications of Machine Learning to Reactor Physics
+
+### Publications
+1. R. Stewart, E. Lum, and C. Pope, “Design of an experimental breeder reactor run 138B reactor	physics benchmark evaluation management application,” Journal of Nuclear Science and Technology, pp. 1-12, 2019, doi:10.1080/00223131.2019.1680325.
```
Comment (nickwhitman1993) [inline users/stewryan/index.md:29]: Extra white space here. Do you want to remove it?

File: users/stewryan/index.md [hunk 2]
```diff
@@ -8,26 +8,32 @@ author: Ryan Stewart
...
+2. R. Stewart, “FRIDGe: Fast Reactor Input Deck Generator,” Journal of Open Source Software, 4 (40), 1489. https://doi.org/10.21105/joss. 01486.
+3. R. Stewart, and T. Palmer, “Geometric Design Space for Sodium Fast Reactors,”	Transactions of 	the American Nuclear Society, vol. 121, 2019, pp. ***-***.
```
Comment (nickwhitman1993) [inline users/stewryan/index.md:31]: White space

**Comment by nickwhitman1993** [comment]
@ryanstwrt Made a few comments. I saw some white space on your page. Other than that, :shipit:

File: users/stewryan/index.md (hunk 2 above)
Comment (aaronjamesreynolds) [reply users/stewryan/index.md:31]: What white space are you talking about? There doesn't appear to be anything extra between lines 30 and 31. I might be missing something.

Comment (nickwhitman1993) [reply users/stewryan/index.md:31]: It may be fine. Lets see

**Pull request merged**